dash/testing/browser.py

```diff
@@ -229,18 +228,17 @@ def take_snapshot(self, name):
 
         self.driver.save_screenshot(f"{target}/{name}_{self.session_id}.png")
 
-    def find_element(self, selector):
-        """find_element returns the first found element by the css `selector`
+    def find_element(self, selector, attribute="CSS_SELECTOR"):
+        """find_element returns the first found element by the attribute `selector`
         shortcut to `driver.find_element(By.CSS_SELECTOR, ...)`."""
```

Comment:
Seems reasonable - let's just describe `attribute` in the docstring (here and `find_elements`) along with the list of acceptable values, which (from `dir(By)`) appears to be:
```
'CLASS_NAME', 'CSS_SELECTOR', 'ID', 'LINK_TEXT', 'NAME', 'PARTIAL_LINK_TEXT', 'TAG_NAME', 'XPATH'
```

Reply:
Added a list of acceptable `attribute` values to the docstring and noted that they align with the Selenium API's `By` class.